experiments/scripts/subj-constants.sh

```diff
@@ -2,95 +2,111 @@
+  #${rootDir}/impact-tools/*:${rootDir}/xml-security-orig-v1/bin/:${rootDir}/xml-security-commons/bin/:${rootDir}/xml-security-orig-v1/data/:${rootDir}/xml-security-commons/libs/*
+  #${rootDir}/impact-tools/*:${rootDir}/ambari/ambari-server/target/classes/:${rootDir}/ambari/ambari-server/target/test-classes/:${rootDir}/ambari/ambari-server/target/dependency/*:${rootDir}/ambari/ambari-server/target/randoop/bin/:/usr/lib/jvm/java-7-oracle/jre/lib/*:
+  #${rootDir}/impact-tools/*:${rootDir}/zeppelin/zeppelin-zengine/target/classes/:${rootDir}/zeppelin/zeppelin-zengine/target/test-classes/:${rootDir}/zeppelin/zeppelin-zengine/target/dependency/*:${rootDir}/zeppelin/zeppelin-zengine/target/randoop/bin/:/usr/lib/jvm/java-7-oracle/jre/lib/*:
+  #${rootDir}/impact-tools/*:${rootDir}/zeppelin/zeppelin-server/target/classes/:${rootDir}/zeppelin/zeppelin-server/target/test-classes/:${rootDir}/zeppelin/zeppelin-server/target/dependency/*:${rootDir}/zeppelin/zeppelin-server/target/randoop/bin/:/usr/lib/jvm/java-7-oracle/jre/lib/*:
+  #/home/user/dependent-tests-impact/experiments/impact-tools/*:/home/user/dependent-tests-impact/experiments/activemq-old/activemq-camel/target/dependency/*:/home/user/dependent-tests-impact/experiments/activemq-old/activemq-camel/target/classes:/home/user/dependent-tests-impact/experiments/activemq-old/activemq-camel/target/randoop/bin:/home/user/dependent-tests-impact/experiments/activemq-old/activemq-camel/target/test-classes:
+  ${rootDir}/impact-tools/*:${rootDir}/xstream-1.2/xstream/target/dependency/*:${rootDir}/xstream-1.2/xstream/target/classes:${rootDir}/xstream-1.2/xstream/target/randoop/bin:${rootDir}/xstream-1.2/xstream/target/test-classes:
 )
 # The name of the subject you want to be displayed in the paper (e.g., XML Security).
 expNameFormal=(
-#  "Crystal"
-#  "JFreechart"
-#  "Joda-Time"
-#  "Synoptic"
-#  "XML Security"
-#  "Ambari-Server"
-#  "Zeppelin-Zengine"
-  "Zeppelin-Server"
+  #"Crystal"
+  #"JFreechart"
+  #"Joda-Time"
+  #"Synoptic"
+  #"XML Security"
+  #"Ambari-Server"
+  #"Zeppelin-Zengine"
+  #"Zeppelin-Server"
+  #"ActivemqCamel"
+  "XStream"
 )
 # Directory for where all of the new subject's information is stored.
 nextExpDirectories=(
-#  ${rootDir}/crystal
-#  ${rootDir}/jfreechart-1.0.16
-#  ${rootDir}/jodatime-d6791cb5f9
-#  ${rootDir}/dynoptic-ea407ba0a750
-#  ${rootDir}/xml-security-1_2_0
-#  ${rootDir}/ambari-new/ambari-server/target
-#  ${rootDir}/zeppelin-zengine-new/zeppelin-zengine/target
-  ${rootDir}/zeppelin-new/zeppelin-server/target
+  #${rootDir}/crystal
+  #${rootDir}/jfreechart-1.0.16
+  #${rootDir}/jodatime-d6791cb5f9
+  #${rootDir}/dynoptic-ea407ba0a750
+  #${rootDir}/xml-security-1_2_0
+  #${rootDir}/ambari-new/ambari-server/target
+  #${rootDir}/zeppelin-zengine-new/zeppelin-zengine/target
+  #${rootDir}/zeppelin-new/zeppelin-server/target
+  #/home/user/dependent-tests-impact/experiments/activemq-new/activemq-camel/target
```

Comment:
Why do you have activemq changes here still? Commit like this will cause merge conflicts with Henry. Neither of you should be modifying the contents of this file. You should only need to add to this file.